kp_test.go

```diff
@@ -369,7 +370,7 @@ func TestKeys(t *testing.T) {
 				testKeys.Metadata.NumberOfKeys++
 
 				MockAuthURL(keyURL, http.StatusCreated, testKeys)
-				k, err := api.CreateImportedRootKey(ctx, "test", nil, payload, "", "")
+				k, err := api.CreateRootKey(ctx, "test", WithPayload(payload, "", "", false))
```

Comment:
Refactored a lot of the methods in this file since i removed a lot of those methods.